### DISCUSSION TITLE
Discrepancy between submitted Score and Replay

### DISCUSSION THREAD

**Discussion by gallantcrow** [ppy/osu#28878] [Q&A] [answered]
Not sure if this is a bug so I'll ask here first.

I currently face the strange issue that a submitted Score on my Leaderboard is not aligning with the local replay within Lazer, which displays a different score and ranking. 

The submitted Score can be found here.
https://osu.ppy.sh/scores/3160146048

I was pretty impressed by my own play so I watched the local replay and it shows the attempt as a perfect attempt with SS-ranking.

Am quite certain that the singular green "OK" Judgement is accurate and that I, in fact, did NOT play this map perfectly.

Do you know what could cause this discrepancy?
Is this a known issue?
If there's anything I can provide to help looking into this, please let me know.

Additional information:
Client: Lazer
OS: Windows 10 Home (22H2)

If Hardware Specs are required, I can submit a DxDiag summary as well.

Screenshots:

Looking at the score of the replay before replaying it:
![image](https://github.com/user-attachments/assets/1f0d6212-7fa4-42dc-9bde-a45a313c974f)

Once the replay has finished:
![image](https://github.com/user-attachments/assets/56cfda45-45d1-4488-8136-e853098072aa)

I could not find any logs under %AppData%/osu/logs, that file-structure just doesn't exist within my appdata folder.

With kind regards, 
DC

Edit:
Added additional information as the original post was lacking details.

**Comment by felipemarins** [comment, marked as answer]
This has been reported several times already: #28368 (in your case, it's probably the first link on that list)
Also, please search for existing issues/discussions before opening a new one.
